src/main/java/com/denizenscript/clientizen/objects/ModTag.java

```diff
@@ -0,0 +1,251 @@
+    //
+    // @Matchable
+    // ModTag matchers:
+    // "mod" plaintext: always matches.
+    // Mod id: matches if the mod id matches the input, using advanced matchers.
+    // -->
+
+    @Fetchable("mod")
+    public static ModTag valueOf(String input, TagContext context) {
+        if (input.startsWith("mod@")) {
+            input = input.substring("mod@".length());
+        }
+        Optional<ModTag> modTag = FabricLoader.getInstance().getModContainer(CoreUtilities.toLowerCase(input)).map(ModTag::new);
+        if (modTag.isEmpty()) {
+            Utilities.echoErrorByContext(context, "valueOf ModTag returning null: '" + input + "' isn't a valid mod name.");
+            return null;
+        }
+        return modTag.get();
+    }
+
+    public static boolean matches(String string) {
+        if (string.startsWith("mod@")) {
+            return true;
+        }
+        return FabricLoader.getInstance().isModLoaded(CoreUtilities.toLowerCase(string));
+    }
+
+    public final ModContainer modContainer;
+
+    public ModTag(ModContainer modContainer) {
+        this.modContainer = modContainer;
+    }
+
+    public ModMetadata getMetadata() {
+        return modContainer.getMetadata();
+    }
+
+    public static void register() {
+
+        // <--[tag]
+        // @attribute <ModTag.id>
+        // @returns ElementTag
+        // @description
+        // Returns a mod's id.
+        // -->
+        tagProcessor.registerStaticTag(ElementTag.class, "id", (attribute, object) -> {
+            return new ElementTag(object.getMetadata().getId(), true);
+        });
+
+        // <--[tag]
+        // @attribute <ModTag.display_name>
+        // @returns ElementTag
+        // @description
+        // Returns a mod's display name.
+        // -->
+        tagProcessor.registerStaticTag(ElementTag.class, "display_name", (attribute, object) -> {
+            return new ElementTag(object.getMetadata().getName(), true);
+        });
+
+        // <--[tag]
+        // @attribute <ModTag.description>
+        // @returns ElementTag
+        // @description
+        // Returns a mod's description.
+        // -->
+        tagProcessor.registerStaticTag(ElementTag.class, "description", (attribute, object) -> {
+            return new ElementTag(object.getMetadata().getDescription(), true);
+        });
+
+        // <--[tag]
+        // @attribute <ModTag.version>
+        // @returns ElementTag
+        // @description
+        // Returns a mod's version.
+        // -->
+        tagProcessor.registerStaticTag(ElementTag.class, "version", (attribute, object) -> {
+            return new ElementTag(object.getMetadata().getVersion().getFriendlyString(), true);
+        });
+
+        // <--[tag]
+        // @attribute <ModTag.authors>
+        // @returns MapTag
+        // @description
+        // Returns a mod's authors, as a map of author name to their contact information.
```

Comment:
Quilt mods will return the author's role in the development (programmer, artist, etc) iirc

Reply:
I mean, this is generally based on the Fabric API & what that returns/is documented as, not sure what Quilt does